### PR TITLE
debugInfo

### PR DESCRIPTION
**Pull request by kalturaguy**
Ability to view player debug information (in future we can put there which HLS segment is downloaded, which one is player, current player buffer size, etc..)

### CODE REVIEW

File: modules/DebugInfo/resources/DebugInfo.js [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+                    internal[name]=_this.$scope[name];
+
+                    function updateHtml() {
+                        var newContent = originalText.replace(match, internal[name]);
+                        el.innerHTML = newContent;
+                    }
+
+                    updateHtml();
+
+                    Object.defineProperty($newscope, name, {
```
Comment (itaykinnrot) [inline modules/DebugInfo/resources/DebugInfo.js]: this will crash on IE8

File: modules/DebugInfo/resources/DebugInfo.js [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                        },
+                        set: function(newValue) {
+                            try {
+                                if (internal[name] !== newValue) {
+                                    internal[name] = newValue;
+                                    updateHtml();
+                                }
+                            }
+                            catch(e) {
+                                alert(e);
```
Comment (itaykinnrot) [inline modules/DebugInfo/resources/DebugInfo.js]: alert?

Comment (kalturaguy) [inline modules/DebugInfo/resources/DebugInfo.js]: Fixed

File: modules/DebugInfo/resources/DebugInfo.js (hunk 1 above)
Comment (kalturaguy) [inline modules/DebugInfo/resources/DebugInfo.js]: Removed support for IE8

**Pull request closed without merging**